portfolio/src/main/webapp/images.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Sibing's image gallery</title>
+        <link rel="stylesheet" href="style.css">
+        <script src="script.js"></script>
+    </head>
+    <body>
+        <div id="content">
+            <h1>Sibing's Image Gallery</h1>
+            <div id="gallery">
+                <a href="images/Sibing.jpg"><img src="images/Sibing.jpg" style="height:150px; width:200px"></a>
```

Comment:
Look like each image has a different naming convention snake, camel, etc. Should we choose one? anything is fine.

Reply:
Fixed